reference/vane-apis/ames.md

```diff
@@ -0,0 +1,303 @@
+there. Otherwise it returns `event-core` unchanged.
+
+Does `%jilt` return any gifts?
+
+
+### %plea
+
+`%plea` is the `task` used to send messages over Ames. It extends the
+`%pass`/`%give` semantics across the network. As such, it is the most
+fundamental `task` in Ames and the primary reason for its existence.
+
+Ames `pass`es a `%plea` `note` to another vane when it receives a message on a
+"forward flow" from a peer, originally passed from one of the peer's vanes to
+peer's Ames.
+
+Ames `pass`es a `%plea` to itself to trigger a heartbeat message to be sent to
+our ship's sponsor
+
+#### Accepts
+
+```hoon
+[=ship =plea]
+```
+
+A `%plea` `task` takes in the `ship` the `plea` is addressed to, and a `$plea`,
+which is
+
+```hoon
+  +$  plea  [vane=@tas =path payload=*]
+```
+Here, `vane` is the destination vane on the remote ship, `path` is the internal
+route on the receiving ship, and `payload` is the semantic message content.
+
+#### Returns
+
+`event-core` is returned, modified to include the received `%plea` (this is not
+a `gift`).
+
+Not sure a `gift` is returned? But if there would be an ack `gift` anywhere, I
+feel like it would be here.
+
+## System and Lifecycle Tasks
+
+### %born
+
+Each time you start your Urbit, the Arvo kernel calls the `%born` task for Ames.
+
+#### Accepts
+
+`%born` takes no arguments.
+
+#### Returns
+
+In response to a `%born` `task`, Ames `%give`s Jael a `%turf` `gift`.
+    
+
+### %crud
+
+`%crud` is called whenever an error involving Ames occurs. It produces a crash
+report in response.
+
+#### Accepts
+
+```hoon
+=error
+```
+
+A `$error` is a `[tag=@tas =tang]`.
+
+#### Returns
+
+Ames does not `give` a `gift` in response to a `%crud` `task`, but it does
+`%pass` Dill a `%flog` `task` instructing it to print `error`. 
+
+
+## %init
+
+`%init` is called a single time during the very first boot process, immediately
+after the [larval stage](@/docs/tutorials/arvo/arvo.md#larval-stage-core)
+is completed. This initializes the vane. Jael is initialized first, followed by
+other vanes such as Ames.
+
+In response to receiving the `%init` `task`, Ames subscribes to the information
+contained by Jael.
+
+#### Accepts
+
+```hoon
+our=ship
+```
+
+`%init` takes in the name of our ship, which is a `@p`. 
+
+#### Returns
+
+```hoon
+    =~  (emit duct %pass /turf %j %turf ~)
+        (emit duct %pass /private-keys %j %private-keys ~)
+```
+
+`%init` returns `event-core` with two new `move`s to be performed that subscribe
+to `%turf` and `%private-keys` in Jael.
+
+
+### %sift
+
+This `task` filters debug output by ship.
+
+#### Accepts
+
+```hoon
+ships=(list ship)
+```
+
+The list of ships for which debug output is desired.
+
+#### Returns
+
+```hoon
+    =.  ships.bug.ames-state  (sy ships)
+    event-core
+```
+
+This `task` returns the `event-core` modified so that debug output is shown only
+for `ships`.
```

Comment:
As in other cases, this the state change, which is interesting, but not part of the public interface.  Unless you intend to describe the state changes?